canonical-web-and-design/vanilla-framework
# Form checkbox and radio button components

## Done

Adds `p-checkbox` and `p-radio` components to have a better control over styling and spacing than with base checkbox styles.

Fixes #2931 
Fixes #3181
Fixes #3177

## QA

-  `./run` or [demo](https://vanilla-framework-3198.demos.haus/docs/examples/patterns/forms/tick-comparison)
- Go to [tick elements comparison page](https://vanilla-framework-3198.demos.haus/docs/examples/patterns/forms/tick-comparison), make sure new components work as expected compared to deprecated base styles
- Check [updated docs about checkbox and radio buttons](https://vanilla-framework-3198.demos.haus/docs/base/forms#checkbox), make sure they are correct: 


## Screenshots

<img width="879" alt="Screenshot 2020-08-05 at 16 52 03" src="https://user-images.githubusercontent.com/83575/89427922-15161600-d73c-11ea-8a7a-95652de848d2.png">
